GOVERNANCE.md

```diff
@@ -146,8 +146,31 @@ Contributions can be:
 
 ### Nominating a new Collaborator
 
-To nominate a new Collaborator, open an issue in the [nodejs/node][] repository.
-Provide a summary of the nominee's contributions. For example:
+To nominate a new Collaborator:
+
+1. **Optional but strongly recommended**: open a
+   [discussion in the nodejs/collaborators][] repository. Provide a summary of
+   the nominee's contributions (see below for an example).
+2. **Optional but strongly recommended**: After sufficient wait time (e.g. 72
+   hours), if the nomination proposal has received some support and no explicit
+   block, add a comment in the private discussion stating you're planning on
+   opening a public issue, e.g. "I see a number of approvals and no block, I'll
+   be opening a public nomination issue if I don't hear any objections in the
+   next 72 hours".
+3. **Optional but strongly recommended**: Privately contact the nominee to make
+   sure they're confortable with the nomination.
+4. Open an issue in the [nodejs/node][] repository. Provide a summary of
+   the nominee's contributions (see below for an example). Mention
+   @nodejs/collaborators in the issue to notify other collaborators about
+   the nomination.
```

Comment:
The public ping can be made optional here if it has been done in the private discussion already:
```suggestion
   the nominee's contributions (see below for an example). Mention
   @nodejs/collaborators in the issue to notify other collaborators about
   the nomination if it hasn't already been done in the private discussion.
```